src/compiler/scala/tools/nsc/CompilerCommand.scala

```diff
@@ -133,7 +131,7 @@ class CompilerCommand(arguments: List[String], val settings: Settings) {
   def expandArg(arg: String): List[String] = {
     def stripComment(s: String) = s takeWhile (_ != '#')
     import java.nio.file._
-    import collection.JavaConverters._
+    import scala.jdk.CollectionConverters._
```

Comment:
Hey, I asked for puns not pricks.

Reply:
Is it too late to pun about Trump complaining about people inviting their relatives to immigrate and help care for grandkids etc? I've forgotten what derogatory word they use for that, but it's not "relative imports".

Reply:
I should have made this and the below comment in the opposite order, so you could have said something about family visas instead.

Reply:
Dang, thirty seconds too slow.

Reply:
@hrhino Just noticed that was a confused emoji, I wasn't wearing my computer glasses before. I meant like Sleeping Beauty pricks her finger on the spindle and "If you prick me, do I not bleed?" spoken IIRC by the guy who wanted a pound of flesh but then was permitted no blood.

Reply:
> What do you have against relative imports?

`collection.JavaConverters` seems so headless to me.  Personally, I wish the magic `import scala._` didn't include packages but just the types in the scala package and the everything in the package object.

However, I contradict myself because I love `sys.error("boom")`.

Reply:
@dwijnand `-Yimports` was made just for you. It used to have an alias `-Wijnand` but I had to remove it because that looks like a warning.

Reply:
If I could `-Yimports:scala\.[A-Z].*'...  But, anyways, I mean as a language default - not just for some projects.